lang/cs/Org.Apache.REEF.Tang/Formats/AvroConfigurationDataContract/AvroConfiguration.cs

```diff
@@ -37,7 +38,7 @@ public AvroConfiguration()
         }
 
         [DataMember]
-        public HashSet<ConfigurationEntry> Bindings { get; set; }
+        public List<ConfigurationEntry> Bindings { get; set; }
```

Comment:
The reason is because AvroSerializer doesn't serialize HashSets correctly and adds empty entries.